go/enclave/rpc/vk_utils.go

```diff
@@ -122,9 +122,11 @@ func withVKEncryption[P any, R any](
 	// 4. Call the function that knows how to validate the request
 	builder := &CallBuilder[P, R]{Status: NotSet, VK: vk, ctx: ctx}
 
+	callName := decodedRequest.Method
+
 	err := validate(decodedRequest.Params, builder, encManager)
 	if err != nil {
-		return responses.AsPlaintextError(errInt), responses.ToInternalError(err)
+		return responses.AsPlaintextError(errInt), responses.ToInternalError(fmt.Errorf("%s: error validating: %w", callName, err))
```

Comment:
this feels like it violates privacy, but at least only in the case of errosr

Reply:
yes. It does. It's hard to understand what failed without it.
It's a compromise